### PR TITLE
Landing page fixes

### PR DESCRIPTION
**Pull request by charan12345678**

### CODE REVIEW

File: app/components/organisms/LitePage/desktop.js [hunk 1]
```diff
@@ -71,20 +71,27 @@ const DesktopPage = ({
...
   };
   const [profession, setProfession] = useState('"Marketing"');
+  const [counter, setCounter] = useState(0);
   const professionArr = allProfessionTypes.map(p => p.name);
   useEffect(() => {
     const visibleProfession =
       professionArr[Math.floor(Math.random() * professionArr.length)];
     const intro = setInterval(() => {
       setProfession(visibleProfession);
+      setCounter(counter + 1);
```
Comment (tuhina2020) [inline app/components/organisms/LitePage/desktop.js:81]: What is the purpose of the counter ? Its not being used anywhere in the code.

**Comment by tuhina2020** [comment]
Please fix the button position in mobile view

File: app/components/NewHeader/index.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { morrflogo as MorrfLogo } from 'Assets/svg-comp';
 import React, { useEffect, useState } from 'react';
 import { getTransitionClass } from 'utils/helper';
 import Button from 'components/molecules/Button';
-
 export default function NewHeader({
   isDesktopOrLaptop,
   bgImg,
+  height,
```
Comment (tuhina2020) [inline app/components/NewHeader/index.js:8]: is the height of header different in mobile and desktop ?

Comment (tuhina2020) [reply app/components/NewHeader/index.js:8]: is there a need to parameterise height ?

**Comment by charan12345678** [comment]
> Please fix the button position in mobile view

Fixed it

**Comment by tuhina2020** [comment]
Fixed it in a different branch. And merged it separately.

**Pull request closed without merging**